static/js/src/stripe/tests/test-responses.js

```diff
@@ -0,0 +1,65 @@
+export const stripeErrorObjects = {
```

Comment:
Maybe this file should be in a `fixtures` directory to make it clear that this is test data rather than a test